data/presets/climbing/route.json

```diff
@@ -0,0 +1,15 @@
+{
+    "icon": "temaki-climbing",
+    "fields": [
+        "name"
+    ],
+    "geometry": [
+        "line",
+        "relation",
+        "point"
```

Comment:
Should we try to migrate those points to #1590 nodes instead? Maybe via MapRoulette…

I wonder, how will a new(ish) user understand what to pick when searching in iD…